### DISCUSSION TITLE
Specify Java version?

### DISCUSSION THREAD

**Discussion by vmiklos** [ReactiveCircus/android-emulator-runner#176] [Q&A] [answered]
Hi,

First, thanks for this great action, I'm using it without problems for a while now.

I recently tried to update to gradle 7, but it requires java 11, and it seems the action provides 1.8. See a sample run here:

https://github.com/vmiklos/plees-tracker/pull/181/checks?check_run_id=3195792409

```
Welcome to Gradle 7.0.2!
...
   > Android Gradle plugin requires Java 11 to run. You are currently using Java 1.8.
```

Is there any way to be able to run gradle 7 with this action?

Thanks,

Miklos

**Comment by ychescale9** [comment, marked as answer]
The default Java version comes from the macos runners. You can easily install and use a different version:
 ```

- name: Java 11
  uses: actions/setup-java@v1
    with:
      java-version: 11
```

**Reply by vmiklos** [reply]
Thanks a lot, that did the trick!
